### DISCUSSION TITLE
Windows messed up

### DISCUSSION THREAD

**Discussion by ganjayin** [valinet/ExplorerPatcher#471] [General]
Hey, I installed this and it literally crashed my windows in a weird way. 

1.) I cannot use the windows start button anymore, its simply not responsive
2.) I cannot search within the file explorer anymore, nothing happens when I click in the search bar and I'm not able to type anything. 
3.) Most annoying: I cannot uninstall it because its not listed in my apps and features list!!

The taskbar issue got fixed whatsoever, but it caused a lot more problems. 
Windows 11 Home - version 21H2 

Please help me fix this mess!! 

Thanks!

**Comment by l0oo** [comment]
_To uninstall, run ep_setup.exe /uninstall_

If not help:
https://www.windowscentral.com/how-reset-windows-11-pc-factory-settings

And just stay with regular Win 11 or back to Win 10.

**Reply by ganjayin** [reply]
Doesnt work, I get this message: 

> Windows cannot find "ep_setup.exe". Make sure you've typed the name correctly, then try again

**Reply by ganjayin** [reply]
I right clicked on the windows logo, selected "run" and pasted that in there. Also tried via task manager - File - run new task.

**Reply by Anixx** [reply]
LOL. Is it first time you use computer? Of course, you have to provide the path to the file.

**Reply by l0oo** [reply]
You need to show Windows path to this file...

**Reply by ganjayin** [reply]
That worked, maybe update your instructions accordingly. Thanks!

**Reply by ganjayin** [reply]
Well, even worse now. I got a message saying that it was uninstalled - but nothing changed. Windows explorer restarted, taskbar was completely black and unresponsive for a while, but it still looks like when it was installed. On top of that, i cant even right-click on the windows logo anymore. Uninstalling this made the problem even worse

**Reply by Anixx** [reply]
Is the taskbar placed vertically? Windows 11 taskbar does not support vertical placement. Did you read README file?

**Reply by l0oo** [reply]
Must be smthng with your Windows, not related to EP, just coincidence.
Restart PC and if not help try maybe reset to factory settings. Make before backup of important files before, of course.
And just stay with regular Win 11 (or back to Win 10).

**Reply by ganjayin** [reply]
Yes I read that file and my taskbar is always placed horizontally at the bottom. I also installed the tool again in order to uninstall again and once again; got uninstall complete message but nothing changes. Explorer restarts and it still looks like the tool is installed. I restarted my laptop, I restarted explorer.exe but nothing helps. 
I also dont have the "right click the taskbar and choose "Properties" option available anymore. Its like the uninstall is not 100% complete and got stuck somewhere

**Reply by ganjayin** [reply]
> Must be smthng with your Windows, not related to EP, just coincidence. Restart PC and if not help try maybe reset to factory settings. Make before backup of important files before, of course. And just stay with regular Win 11 (or back to Win 10).

Coincidence? The taskbar looks like EP is installed and after uninstalling it, it still looks like that windows remains messed up? I highly doubt its a coincidence

**Reply by valinet** [reply]
Rename dxgi.dll to dxgio.dll manually from C:\Windows if it’s still there and reboot. If issue persists, it’s not due to EP.

**Reply by ganjayin** [reply]
> Rename dxgi.dll to dxgio.dll manually from C:\Windows if it’s still there and reboot. If issue persists, it’s not due to EP.

Doesnt help either. I think the uninstaller is simply not working because my taskbar items are still being ungrouped. When I run the uninstaller for the first time, I get a success message, but nothing happens. When I run it for a 2nd time, this message pops up (uploaded on imgbb.com) https://ibb.co/PQ8bbSf

The troubleshooting steps in there dont help either.

- Properties dialogue is not open, I dont even know which dialogue this is referring to
- Kill and restart all explorer.exe processes didnt do anything
- Rebooting the computer doesnt do anything either.

And this is 100% caused by EP. I did not have any of these issues before. They immediately occurred after the installation and most importantly, the taskbar items are still ungrouped (which means that EP still must be installed somewhat). Hence uninstalling it doesn't work despite getting a "success" message.

**Reply by valinet** [reply]
No, it helps if you do it. EP cannot affect explorer if it is not loaded into it. Make sure you have no dxgi.dll in C:\Windows and that’s it. If you still have issues then, it’s something else.

**Reply by ganjayin** [reply]
I removed it and restarted, nothing changed. If this is not EP related, can you please explain why my taskbar buttons are still uncombined? There is no other way to un-combine them and the ONLY workaround I tried so far was installing EP. It worked immediately after installing EP and changing the settings as per the guide, and thats also when the problems started. I just think that the uninstaller didnt uninstall properly, otherwise the taskbar would go back to its initial stage (automatically combining all items), no?

**Reply by valinet** [reply]
You think what you want, I know how it works. If it is not loaded in Explorer, how can it still affect it? As I said, if you removed it from C:\Windows, it should not be loaded by other means. To make sure, you can use Process Explorer to check out the DLLs loaded by Explorer. If the dxgi from C:\Windows is in there, you probably haven’t removed it. Also, if you registered it as shell extension, check and make sure ExplorerPatcher.amd64.dll is not in that list. I am pretty sure though removing it from C:\Windows should be enough in your case. There are other ways to enable the ungrouped taskbar, like the UndockingDisabled registry entry etc. Try to take this logically, understand the advice I gave you and determine the cause. Continuously blaming the same thing doesn’t help at all.

**Reply by ganjayin** [reply]
I'm not blaming anything or anyone, I'm asking simple questions in order to fix my issue and the only place to get help, is here. I'm following the advice and I'm reporting back what happens, so I don't understand your attitude unless you get offended easily when things dont work out your way. 

Again, I have not taken any other steps other than installing this tool (yesterday), in order to uncombine my taskbar items and I dont even have the knowledge to do the "UndockingDisabled registry entry" or any other steps. I can confirm 1000% that the mentioned items are removed and I can send you screenshots if you have trust issues. 

Hence I'm taking this "logically" by explaining you my steps and the "cause" is clear: the EP tool is not uninstalling properly, otherwise the taskbar items would be combined again, as per Windows 11 default settings. What else should cause my taskbar items to remain uncombined? There is nothing else because I have not taken any other steps other than installing this tool yesterday and uninstalling it today.

**Reply by ganjayin** [reply]
Proof that both DLLs are not loaded/running/existing: 
https://ibb.co/jG3Wkvz
https://ibb.co/5FZvdP1

**Reply by valinet** [reply]
> Proof that both DLLs are not loaded/running/existing: https://ibb.co/jG3Wkvz https://ibb.co/5FZvdP1

That's not what I said above. I said to look whether `dxgi.dll` or `ExplorerPatcher.amd64.dll` is loaded in `explorer.exe`, not to randomly look for some DLL in the processes list, it will never be there. As I said, try to follow some logic. DLLs are not running by themselves, they are hosted by some process that runs an executable. You search for `explorer.exe`, you click it, go to View -> Show Lower Pane, View -> Lower pane View -> DLLs, you look those in that list that shows below the process tree. If they're not there, then as far as it's concerned, it is uninstalled, so it might be something else, which I don't really know to tell you what might be. Maybe Microsoft has some guide to reset the new taskbar to its defaults, search for that, idk about one...

**Comment by ganjayin** [comment]
Thank you for the additional tips and instructions, the DLLs are not listed there either, or they would appear in this alphabetical order: https://ibb.co/rM48yXF (I also went through the whole list manually, just in case) 

It's just very strange that these issues (Windows button in taskbar irresponsive to left and right-click, as well as the irresponsive search function in any windows explorer window) occurred right after the installation (which is why I reached out here to get it fixed) and remain, even though it seems to be completely uninstalled. And in addition to that, the taskbar items are still ungrouped, which I took as an indicator that the uninstall was not fully complete.

Would you know by any chance how to fix the above (Windows button/Windows explorer issue) when EP is installed? Or are they known issues/bugs? 

Otherwise, I will have to find other solutions and leave it uninstalled, and most likely make use of the factory reset, which I tried to avoid. 

Thanks!

**Comment by matzoshaina** [comment]
check the following registry setting
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell\Update\Packages

be sure there is no dword for "UndockingDisabled"

if so, delete it and reboot

**Reply by ganjayin** [reply]
Thank you, that fixed all the issues. Much appreciated!! 🙏

**Reply by Anixx** [reply]
So, it is not related to EP. You did it yourself and forgot.

**Reply by Anixx** [reply]
And, the issues you listed in the original post, are mostly because of this registry entry either, not due to EP.

**Reply by ganjayin** [reply]
> So, it is not related to EP. You did it yourself and forgot.

Thats not true, I dont even know how to do those things and I never edited the registry on this computer (until I followed todays advice). This issue started immediately after installing EP and I think if I knew how to "implement" this, I would have known how to fix this also. Maybe changing taskbar settings unknowingly did this, but I did not knowingly or purposely make any changes like that.

**Reply by Anixx** [reply]
EP does not affect this registry entry. Neither does any built-in Windows dialog.

**Reply by valinet** [reply]
EP does not touch that registry entry. I mentioned it above as well, not as verbose though. Some program that you used enabled that. As I said, stop needlessly blaming EP on this one, even though it seemed like it did something, I gave you all the rational arguments about how it did not do and doesn’t do anything. Its patches are mostly in memory, specifically to avoid UndockingDisabled disabled, which is not recommended, and has all types of side effects, EP was specifically designed to avoid the need of using that; I don’t know what the exact purpose of this registry entry is, probably to enable the half broken legacy mode in Windows Team where the interface is totally different anyway. An in memory patch cannot affect some other program without being loaded in memory, and enabling the old taskbar is an in-memory patch done every time Explorer starts up when EP is loaded.

So, again, EP does not touch that, if you don’t believe it, “grep” all the source for that or run “strings64” on the binary, you won’t find a mention of it at least, so how can it change it without using its name…? It was just a coincidence. Better to find out what exactly changed that. This is yet again advice on why it is best not to run this alongside a million other tweaks, speaking in general.

Anyway, I am glad that you sorted it out.

**Reply by ganjayin** [reply]
Once again, I'm not blaming EP, I'm just REPORTING step by step what is happening and what has happened. Basic troubleshooting and reporting in order to find a solution with the creator. I'm no expert in this field at all. I searched for a solution on how to fix this grouping issue, came across your tool, and the reported issues started immediately after the installation. I have not taken any other steps to possibly modify the taskbar or its behavior prior to installing EP (other than checking the regular Windows taskbar settings). That's all. After all, this could also be a learning opportunity IN CASE something might be wrong or associated with my windows version. No one and nothing gets blamed, so please stop blaming me for "blaming" EP, thank you! 

Have a great weekend!

**Comment by Andre-10** [comment]
Wait, in the 1st post you stated Windows version as "Windows 11 Home - version 21H2"

There is no 21H2 for Windows 11. 21H2 is Windows 10 version. Do you run Windows 11, or 10?

**Reply by ganjayin** [reply]
Not sure what you're trying to say but this is from my Windows 11 settings: 

Edition	Windows 11 Home
Version	21H2
Installed on	‎21/‎10/‎2021
OS build	22000.318
Experience	Windows Feature Experience Pack 1000.22000.318.0

[screenshot](https://ibb.co/DMCSvtX)

Also, official [Microsoft documentation](: https://docs.microsoft.com/en-us/windows/release-health/windows11-release-information) lists this version

**Comment by Andre-10** [comment]
Sorry, my bad.

I'm trying to understand what is going on with your install. 

Valinet explained that there's no (OK, very slim) chance about EP causing that.

If you don't have any other tweaker installed (Winaero maybe) to help you undo problematic behavior, I would do a repair install with the latest Windows 11 ISO

**Reply by ganjayin** [reply]
No worries! 

I really haven't installed or tweaked anything else. In fact, this laptop only recently updated to W11. There are only standard apps such as Office, Skype, Spotify, Slack etc. installed as this is my work laptop. I also noticed that shortly after the 1st installation, the additional "properties" function shortly disappeared when right-clicking my taskbar. 

So after removing everything successfully, I tried the installation a 2nd time and the same issues occurred again. Then I removed everything again and hoped to be third time lucky. After the 3rd installation, it finally worked. Now the only issue is that I can't click on the battery status icon (well i can click on it, then the windows loading circle pops up for a split second, and then nothing happens). All the other icons such as wifi, speaker etc work fine. Thats just a minor issue not bothering me at all, so I'm happy that everything is fine now. 

Really not sure what happened but I'm not a programmer or anything like that. I know some bits and bobs from when I worked in IT over 10 years ago, but that's about it :)

**Reply by Andre-10** [reply]
Well. Important is that it's solved. 

With repeated installations. I'm guessing, but antivirus or something similar may be involved, preventing successful installation. Have no idea, but your post could help someone with similar problems

**Comment by matzoshaina** [comment]
with regards to the battery flyout, open the properties for EP, on the left, click on  system tray. Note on the right near the bottom of the list is an entry for battery. Click on the default windows 10 flyout and change it to windows 7
